explcheck/src/explcheck.lua

```diff
@@ -0,0 +1,168 @@
+local common = require("explcheck-common")
```

Comment:
Perhaps rename `explcheck.lua` to `explcheck-cli.lua` to make it clear that it is not the only or preferred entry point.

Reply:
Fixed in commit 1e4d8a5 from this PR.